hw/ip/otbn/data/base-insns.yml

```diff
@@ -372,6 +372,8 @@
     The initial value in `grs1` is treated as a bit mask that specifies bits to be set in the CSR.
     Any bit that is high in `grs1` will cause the corresponding bit to be set in the CSR, if that CSR bit is writable.
```

Comment:
The "if that CSR is writable" part of the sentence should probably go and be combined with the text below, since it doesn't really specify what happens if the CSR happens to be not writable.